src/Vanguard/VanguardClient.cpp

```diff
@@ -21,10 +21,10 @@
 #using < system.collections.dll>
 
 //If we provide just the dll name and then compile with /AI it works, but intellisense doesn't pick up on it, so we use a full relative path
-#using <../../../../../RTCV/Build/NetCore.dll>
-#using <../../../../../RTCV/Build/Vanguard.dll>
-#using <../../../../../RTCV/Build/CorruptCore.dll>
-#using <../../../../../RTCV/Build/RTCV.Common.dll>
+#using <../../../RTCV/Build/NetCore.dll>
+#using <../../../RTCV/Build/Vanguard.dll>
+#using <../../../RTCV/Build/CorruptCore.dll>
+#using <../../../RTCV/Build/RTCV.Common.dll>
```

Comment:
This may break your existing build workflow. However, both in CI and locally, this works with a directory structure like:
```
Code\
    RTCV\
    melonDS-Vanguard\
```